language/types_values_variables.md

```diff
@@ -646,6 +978,28 @@ as negative value.
 
 Defaults to `p`
 
+##### Timespan to String
+
+| Format  | Timespan formats
+| ----    | ------------------
+| s       | formats according to the timespan format string '%D-%H:%M:%S' 
+| p       | programmatic representation - a quoted string formatted according to the timespan format string '%D-%H:%M:%S'
```

Comment:
I guess we should add similar entries for SemVer('str') and SemVerRange('str').  I think the String conversion of those types is missing altogether.